src/libs/Clipboard/index.js

```diff
@@ -29,6 +46,24 @@ function setHTMLSync(html, text) {
     document.body.appendChild(node);
 
     const selection = window.getSelection();
+    let originalSelection = null;
+    const firstAnchorChild = selection.anchorNode && selection.anchorNode.firstChild;
```

Comment:
I'm not sure that checking for `selection.anchorNode` is needed here, it is only null if we have no selection, and this function is only triggered when we have a selection.

Reply:
It will be null if we are using copy button as described by @mollfpr [here](https://github.com/Expensify/App/issues/14300#issuecomment-1398415926).

Reply:
Yeah if you notice the selection is gone when we right-click, as I described [here](https://github.com/Expensify/App/pull/14506#issuecomment-1402077502). So if we fix the right-click issue we won't need the check.